src/engine.jl

```diff
@@ -21,6 +21,8 @@ type MSession
 
         ep = ccall(engfunc(:engOpen), Ptr{Void}, (Ptr{UInt8},), default_startcmd)
         ep == C_NULL && throw(MEngineError("Failed to open a MATLAB engine session."))
+        # hide the MATLAB command window on Windows
```

Comment:
Why is this windows only? (And there's no way to create a hidden session directly?)